utils_linux.go

```diff
@@ -171,6 +171,9 @@ func createPidFile(path string, process *libcontainer.Process) error {
 	if err != nil {
 		return err
 	}
+	if _, err := os.Stat(path); err == nil {
+		return fmt.Errorf("file %s already exists", path)
```

Comment:
yah overwriting pid files is usually a norm. The easy way here is probably just detect at the flag parsing time (i.e. socket file is not the same as pid file )